cli/diagnostics.rs

```diff
@@ -29,6 +29,7 @@ const UNSTABLE_DENO_PROPS: &[&str] = &[
   "Location",
   "MXRecord",
   "Metrics",
+  "NAPTRRecord",
```

Comment:
Hmmm, this shouldn't be needed... `Deno.resolveDns()` is not stable API. We should remove all of the types related to that API from this list

Reply:
Opened https://github.com/denoland/deno/pull/14618

Reply:
Ah oops! Good catch 😄